js/src/Ice/InstanceExtensions.js

```diff
@@ -181,6 +182,12 @@ Instance.prototype.finishSetup = function (communicator) {
             this._initData.properties.getIcePropertyAsInt("Ice.Connection.Client.InactivityTimeout"),
         );
 
+        if (FileLogger !== null && this._initData.logger === null) {
```

Comment:
Maybe we should read the property first and show an exception if we're in the browser and the user is trying to use the file logger.